client/index.spec.js

```diff
@@ -0,0 +1,15 @@
+import ReactDOM from 'react-dom';
+import AppRender from './index';
+
+describe('Root component test', () => {
+    beforeEach(() => {
+        window.document.getElementById = () => true;
+        ReactDOM.render = jest.fn();
+    });
+
+    it('should call ReactDOM.render', () => {
+        AppRender();
+
+        expect(ReactDOM.render).toHaveBeenCalled();
```

Comment:
What if I told you that you can check arguments which this function has been invoked with?)
In particular, you can find the first argument and make snapshot from rendered React element!

Reply:
Additionally, `renderApp` method is already being called inside `client/index.js`